backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/generics/report_logo.jrxml

```diff
@@ -18,8 +19,8 @@
 	<field name="?column?" class="java.lang.Integer"/>
 	<title>
 		<band height="85" splitType="Stretch">
-			<image scaleImage="RetainShape" hAlign="Right">
-				<reportElement uuid="8a7336c5-5f63-41d5-8edd-dfc316734e2b" key="image-1" x="356" y="2" width="217" height="79"/>
+			<image scaleImage="RetainShape" hAlign="Left">
+				<reportElement key="image-1" x="378" y="0" width="217" height="79" uuid="8a7336c5-5f63-41d5-8edd-dfc316734e2b"/>
```

Comment:
Like above!
Do not change this subreport.
I will affect all docs. You shall do adjustments from your own report.